RotationControl/src/main/res/values/arrays.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string-array name="scope">
+        <item>com.google.android.youtube</item>
+        <item>com.netflix.mediaclient</item>
+        <item>org.jellyfin.mobile</item>
```

Comment:
I've added these as defaults, because I think that this is most useful for  video entertainment apps.

Reply:
:+1: